Author: Ralf Treinen <treinen@debian.org>
Description: Add to the man page pointer to the complete documentation in html.

--- trunk~/doc/ocamlweb.1	2005-11-04 14:30:08.000000000 +0100
+++ trunk/doc/ocamlweb.1	2005-11-05 13:17:17.839779033 +0100
@@ -28,6 +28,10 @@
 .TP
 .B \-h
 Help. Will give you the complete list of options accepted by ocamlweb.
+For detailed explanations, please check the
+ocamlweb manual.  On a Debian system the manual can be found at
+.I /usr/share/doc/ocamlweb
+in HTML format, or via the Debian help system.
 
 
 .SH AUTHORS
